### PR TITLE
feat(Uwowocosplay): fix presence for en-de paths

### PR DESCRIPTION
**Pull request by PrincessAkira**
## Description 

Presence was strangely broken and then randomly worked.
Made it persistent, since i initially hated the way i did it in the first place. Yall merged faster than i could fix it today haha

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
Before when opening specific cosplay:
image

and Now
image

</details>

### CODE REVIEW

**Comment by theusaf** [comment]
> Oh wait. I did update the Repo. All the other commits still here. Should i have had it deleted before and re-cloned? If yes i can do that no problem here.

That would be preferable. However, it isn't 100% necessary.

File: websites/R/Rate Your Music/metadata.json
Comment (theusaf) [inline websites/R/Rate Your Music/metadata.json:1]: Ensure to undo changes to this file, as it is unrelated to the main changes in this PR

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/R/Rate Your Music/metadata.json
Comment (PrincessAkira) [reply websites/R/Rate Your Music/metadata.json:1]: Yeah. I just noticed everytime i push smth it wants me to increment this .=.

Comment (PrincessAkira) [reply websites/R/Rate Your Music/metadata.json:1]: Also i havent changed this file wtf.
Lemme re-clone this repo tomorrow and make a clean PR :D

**Pull request closed without merging**